src/renderer/pages/UploaderConfigPage.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div id="config-list-view">
+    <el-row :gutter="20" align="middle" class="config-list">
+      <div class="view-title">
+        {{ $T('SETTINGS') }}
+      </div>
+      <el-col
+        class="config-item-col"
+        v-for="item in curConfigList"
+        :key="item._id"
+        :span="11"
+        :offset="1"
+      >
+        <div
+          :class="`config-item ${defaultConfigId === item._id ? 'selected' : ''}`"
```

Comment:
Just highlighting does not give the users a hint that this is the current selection, it is better to give a clear text hint